docs/src/languages/elixir.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Elixir"
+weight: 2
+description: Platform.sh supports building and deploying applications written in Elixir. There is no default flavor for the build phase, but you can define it explicitly in your build hook. Platform.sh Elixir images support both committed dependencies and download-on-demand. The underlying Erlang version is 22.0.7.
+---
+
+{{< description >}}
+
+## Supported versions
+
+{{< image-versions image="elixir" status="supported" >}}
+
+To specify an Elixir container, use the `type` property in your `.platform.app.yaml`.
+
+{{< readFile file="src/registry/images/examples/full/elixir.app.yaml" highlight="yaml" >}}
+
+## Assumptions
```

Comment:
We don't have an "Assumptions" section in any other language, nor do we mention the build flavor.  Only PHP and node have build flavors; they're the odd-balls.  For other languages just say nothing.